sao.js

```diff
@@ -95,6 +98,9 @@ module.exports = {
       default: 'npm'
     }
   },
+  data: {
+    edge: Boolean(argv.edge)
```

Comment:
Can't we simply use `process.argv.includes('--edge')`? :D

Reply:
You right!

Never forget the [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) 😅